docker-compose.yml

```
@@ -1,62 +1,37 @@
 version: '3.9'
 services:
   cache:
-    container_name: backend_cache
-    image: redis:6.2.5-alpine
-    command: redis-server --appendonly yes
-    volumes:
-      - cache_data:/var/lib/redis/data
+    image: redis:alpine
+    restart: on-failure
     ports:
-      - '6379:6379'
+      - '6380:6379'
+    volumes:
+      - cache:/data
 
   db:
-    container_name: backend_db
     image: postgres:latest
     volumes:
-      - postgres_data:/var/lib/postrgresql/data/
+      - db:/var/lib/postrgresql/data/
     ports:
-      - '5432:5432'
+      - '5433:5432'
```

same question?

```suggestion
      - '5432:5432'
```

if you are encountering errors because the port is already taken on you docker machine, you should stop the running containers. e.g. 

```shell
docker stop $(docker ps -a -q)
```